src-java/kilda-model/src/main/java/org/openkilda/model/GroupId.java

```diff
@@ -22,14 +22,14 @@
 import java.io.Serializable;
 
 @Value
-public final class GroupId implements Comparable<GroupId>, Serializable {
+public class GroupId implements Comparable<GroupId>, Serializable {
     private static final long serialVersionUID = 1L;
 
     public static final GroupId MIN_FLOW_GROUP_ID = new GroupId(2);
```

Comment:
Shoiuld we mention here already existing (used now) group ids? Just to avoid possible reusage.

Reply:
The constant used now has been added.